docs/source/releasenotes.rst

```diff
@@ -5,6 +5,10 @@ Release notes
 `Upcoming release <https://github.com/robocorp/rpaframework/projects/3#column-16713994>`_
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
+- Library **RPA.Assistant** 2.3.0:
+
+  - Add new elements with `Add Loading Spinner` and `Add Loading Bar`.
```

Comment:
```suggestion
- Library **RPA.Assistant** 2.3.0 (:pr:`975`):

  - Add new elements with ``Add Loading Spinner`` and ``Add Loading Bar``.
```

At least a PR/Issue should be linked and the keywords are doubled in backticks, otherwise they get rendered in italic (not code) text.

See all the things to be taken into account on [Release notes formatting](https://github.com/robocorp/rpaframework/blob/master/docs/source/contributing/development.md#release-notes-formatting)